nuntium/templates/nuntium/profiles/stats.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base_edit.html" %}
+{% load i18n %}
+{% load pagination_tags %}
+{% load nuntium_tags %}
+{% load staticfiles %}
+
+{% block extrajs %}
```

Comment:
There's no need to include the block if you're not putting anything in it and not overriding anything.